client/src/components/Home.js

```diff
@@ -4,22 +4,27 @@ import Button from '@material-ui/core/Button';
 import Icon from '@material-ui/core/Icon';
 import Grid from '@material-ui/core/Grid';
 import { withStyles } from '@material-ui/core/styles';
+import Leaderboard from './Leaderboard';
+import { getTopFive } from '../utils/leaderboad-api';
 
 const styles = theme => ({
   button: { margin: theme.spacing.unit },
   rightIcon: { marginLeft: theme.spacing.unit },
 });
 
 const Home = ({ classes }) => (
-  <Grid container direction="row" justify="center" alignItems="center">
-    <Button className={classes.button} variant="contained" color="primary" size="large">
-      Play
-      <Icon className={classes.rightIcon}>send</Icon>
-    </Button>
-    <Button className={classes.button} variant="contained" color="secondary" size="large">
-      Find a Friend
-    </Button>
-  </Grid>
+  <React.Fragment>
+    <Grid container direction="row" justify="center" alignItems="center">
+      <Button className={classes.button} variant="contained" color="primary" size="large">
+        Play
+        <Icon className={classes.rightIcon}>send</Icon>
+      </Button>
+      <Button className={classes.button} variant="contained" color="secondary" size="large">
+        Find a Friend
+      </Button>
+    </Grid>
+    <Leaderboard getTopFive={getTopFive} />
```

Comment:
getTopFive, here 5 can be a variable, use this as constant so that we can change this whenever we required, same on the server side.

Reply:
Okay, I will make this change.

Reply:
I will make this change on the client side in this PR as `getTopPlayers`. It will get whatever number of players the server sends and display it. I will change the server side code in another PR. Is that fine? @nimish-gupta